src/pages/saude_em_ordem.py

```diff
@@ -0,0 +1,481 @@
+        yield new_index
+        size = size - math.ceil(size / n)
+        last = new_index
+        n = n - 1
+
+
+def range_separators_indexes(values, n):
+    # Values must be ordered from lowest to highest
+    separations = [
+        (((values[-1] - values[0]) / (n)) * (order + 1)) + values[0]
+        for order in range(n - 1)
+    ]
+    # print(separations)
+    return [
+        bisect.bisect_right(values, separationvalue) for separationvalue in separations
+    ]
+
+
+def chunks_by_range(target_list, key, n):
+    # For use in a list of dicts and to sort them by a specific key
+    split_indexes = range_separators_indexes(
+        [element[key] for element in target_list], n
+    )
+    chunks = []
+    last = 0
+    for i in range(n - 1):
+        chunks.append(target_list[last : split_indexes[i]])
+        last = split_indexes[i]
+    chunks.append(target_list[last::])
+    return chunks
+
+
+def convert_money(money):
+    """ Can be used later to make money look like whatever we want, but a of
+        now just adding the decimal separator should be enough
+    """
+    return f"{int(money):,}".replace(",", ".")
+
+
+def gen_sorted_sectors(sectors_data, slider_value, by_range=True):
+    column_name = "cd_id_" + "%02d" % (int(slider_value / 10))
+    kept_columns = [
+        "cnae",
+        "n_employee",
+        "security_index",
+        "total_wage_bill",
+        "sector",
+        column_name,
+        "activity_name",
+    ]
+    data_for_objects = sectors_data[kept_columns]
+    data_for_objects = data_for_objects.rename(columns={column_name: "score"})
+    sectors = data_for_objects.to_dict(orient="records")
+    sectors.sort(key=lambda x: x["score"])
+    if by_range:
+        sector_groups = chunks_by_range(sectors, "score", 4)
+    else:
+        sector_groups = list(chunks(sectors, 4))
+    return sector_groups
+
+
+def main(user_input, indicators, data, config, session_state):
+    st.write(
+        '<meta name="viewport" content="width=device-width, initial-scale=1.0">',
+        unsafe_allow_html=True,
+    )
+    if (
+        session_state.saude_ordem_data == None
+    ):  # If not loaded, load the data we are going to use in the user database
+        session_state.saude_ordem_data = {
+            "slider_value": 70,
+            "opened_tables": [False, False, False, False],
+            "opened_detailed_view": False,
+        }
+    score_groups, economic_data = get_score_groups(config, session_state)
+    # gen_header()
+    gen_intro()
+    gen_illustrative_plot(score_groups, session_state)
+    gen_slider(session_state)
+    gen_detailed_vision(economic_data, session_state)
+    gen_sector_tables(session_state, score_groups, default_size=5)
+
+
+def get_score_groups(config, session_state):
+    """ Takes our data and splits it into 4 sectors for use by our diagram generator """
+    uf_num = utils.get_place_id_by_names(session_state.state)
+    CNAE_sectors = loader.read_data(
```

Comment:
need to change here to drive_paths: https://github.com/ImpulsoGov/farolcovid/blob/master/src/configs/config.yaml#L17